src/multibody/liegroup/special-orthogonal.hpp

```diff
@@ -475,7 +475,8 @@ namespace pinocchio
                                      const Eigen::MatrixBase<JacobianIn_t> & Jin,
                                      const Eigen::MatrixBase<JacobianOut_t> & J_out) const
     {
-      typedef typename SE3::Matrix3 Matrix3;
+      typedef typename Config_t::Scalar Scalar;
+      typedef typename SE3Tpl<Scalar,PINOCCHIO_EIGEN_PLAIN_TYPE(Config_t)::Options>::Matrix3 Matrix3;
```

Comment:
The class this function belongs to is templatized by a scalar. Why don't we typedef a SE3 at the class level instead of in each function ?

The original typedef came from the non-scalar templetized first version I wrote.

Reply:
Makes sense. Thanks for the insight.